packages/flutter/lib/src/material/navigation_bar.dart

```diff
@@ -422,9 +422,11 @@ class NavigationDestination extends StatelessWidget {
         return Padding(
           padding: const EdgeInsets.only(top: 4),
           child: MediaQuery.withClampedTextScaling(
-            // Don't scale labels of destinations, instead, tooltip text will
-            // upscale.
-            maxScaleFactor: 1.0,
+          // Set maximum text scale factor to 1.3 for the
```

Comment:
Can we make this 1.3 a constant and put it on the top of the file? And also wonder if there's any link that we can refer for this number?

nit: should be 2-character indentation:)